### PR TITLE
fix: update spring boot version to 2.7.0

### PR DESCRIPTION
**Pull request by igdianov**
- [x] Update Spring Boot to 2.7.0
- [x] Update Spring Cloud to 2021.0.2 
- [x] Remove the override of spring-cloud-function 
- [x] Pin H2 database dependency version to 1.4.200 due to breaking changes for now 
- [x] Pin graphql-java version to 13.0 due to breaking changes in managed 18.1 version

Part of https://github.com/Activiti/Activiti/issues/3949

### CODE REVIEW

File: activiti-cloud-build/activiti-cloud-build-dependencies-parent/pom.xml [hunk 1]
```diff
@@ -16,9 +16,21 @@
     <spring-cloud-function.version>3.2.3</spring-cloud-function.version>
```
Comment (erdemedeiros) [inline activiti-cloud-build/activiti-cloud-build-dependencies-parent/pom.xml]: Could you upgrade `spring-cloud.version` to version `2021.0.2`, so we can remove the override of `spring-cloud-function`?

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
Hi Igor,
Unless you prefer to do it into a dedicated PR, let's upgrade the version of Spring Cloud as well.

File: activiti-cloud-build/activiti-cloud-build-dependencies-parent/pom.xml (hunk 1 above)
Comment (igdianov) [reply activiti-cloud-build/activiti-cloud-build-dependencies-parent/pom.xml]: Done

**Review by andrea-ligios — APPROVED** [review]
LGTM!

**Review by erdemedeiros — APPROVED** [review]
LGTM!

**Pull request merged**